cli/src/app.rs

```diff
@@ -59,7 +59,31 @@ pub fn app() -> Command {
                 .help("Reduce the level of verbosity (the maximum is -qq)")
                 .action(ArgAction::Count)
                 .conflicts_with("verbose"),
-        ])
+        ]);
+
+    app = add_subcommands(app);
+
+    app = extensions::add_extensions_subcommands(app);
+
+    #[cfg(feature = "selfmanage")]
+    {
+        app = app.subcommand(
+            Command::new("uninstall").about("Uninstall the Phylum CLI").arg(
+                Arg::new("purge")
+                    .action(ArgAction::SetTrue)
+                    .short('p')
+                    .long("purge")
+                    .help("Remove all files, including configuration files (default: false)"),
+            ),
+        );
+    }
```

Comment:
Does this mean an extension named `uninstall` will be allowed?

Reply:
Hm I guess so. Should probably move this to the `add_subcommands` function.